plutus-tx/src/PlutusTx/Builtins/Internal.hs

```diff
@@ -822,3 +823,16 @@ replicateByte n w8 =
       Haskell.error "byteStringReplicate errored."
     BuiltinSuccess bs -> BuiltinByteString bs
     BuiltinSuccessWithLogs logs bs -> traceAll logs $ BuiltinByteString bs
+
+{-# NOINLINE modularExponentiation #-}
```

Comment:
I think this should probably be exported from [PlutusTx.Builtins](https://github.com/IntersectMBO/plutus/blob/f74023ec043bd28d4fd16008f61b8d5d138cc73f/plutus-tx/src/PlutusTx/Builtins.hs#L128) as well (I've been caught out by that before).  Most of the other builtins have to have a wrapper to handle the `Builtin*` types in that file as well, but I think you can get away without that if everything is an `Integer` (maybe see [this comment](https://github.com/IntersectMBO/plutus/blob/f74023ec043bd28d4fd16008f61b8d5d138cc73f/plutus-tx-plugin/src/PlutusTx/Compiler/Builtins.hs#L99)?).

Reply:
Maybe it also has to go in [PlutusTx.Prelude](https://github.com/IntersectMBO/plutus/blob/master/plutus-tx/src/PlutusTx/Prelude.hs).  I'm always confused by this.